Manager/TicketManager.php

```diff
@@ -39,7 +39,7 @@ public function __construct(EntityManagerInterface $manager, string $class, stri
      * @throws \Doctrine\ORM\OptimisticLockException
      */
     public function createTicket(UserInterface $user, Ticket $ticket){
-        $status = $this->ticketStatusManager->getRepository()->findOneBy(['name' => $this->defaultStatusName]);
+        $status = $this->ticketStatusManager->getRepository()->findOneBy(['name' => $this->ticketStatusManager->getDefaultStatus()]);
```

Comment:
Replace this line like this :  

``status = $this->ticketStatusManager->getDefaultStatus();``